dev.sh

```diff
@@ -16,7 +16,7 @@ for DIR in "$plugindir"/wp-modules/*; do
 		fi
 
 		# Run the build script for this module.
-		npm run dev &
+		npm ci && npm run dev &
```

Comment:
This took about 30 seconds, even the 2nd time.

It looks like doing `npm ci` from within `wpps-scripts` takes a lot longer than doing it from `plugins/pattern-manager/`

Reply:
Huh! I wonder why. Maybe instead of this we'll just make a separate command like npm run ci, that loops through each module and runs ci so we don't have to manually cd into each wp-module.

Reply:
Sure, that might be faster.

Reply:
I removed this to focus this PR on just loading eslint from plugin if it exists